packages/cli/test/unit/commands/__snapshots__/help.test.ts.snap

```diff
@@ -2,38 +2,24 @@
 
 exports[`help command > alias help output snapshots > alias help column width 40 1`] = `
 "
-  ▲ vercel alias [command] [options]
+  ▲ vercel alias [command]
 
-  Interact with deployment aliases.     
+  Interact with deployment aliases      
 
   Commands:
 
-  ls                     Show   
-                         all    
-                         aliase…
-  set  deployment alias  Create 
-                         a new  
-                         alias  
-  rm   alias             Remove 
-                         an     
-                         alias  
-                         using  
-                         its    
-                         hostna…
-
-
-  Options:
```

Comment:
I would feel more comfortable making this help change after we've added help output for the subcommands, which have these options.

Reply:
Two options:

1. Wait for the subcommand help before merging this.
2. Put these options back at the root level, so that they will remain included in the help output. This means we'll have to define no-op telemetry methods on the root-level `AliasTelemetryClient` so that it satisfies the type interface.

Reply:
2. might be able to be considered not a semver major change as well.